### PR TITLE
Update raiden-contracts to 0.9.0

### PR DESCRIPTION
**Pull request by palango**
Small step in preparation for the services integration.

### CODE REVIEW

File: constraints.txt [hunk 1]
```diff
@@ -92,7 +92,7 @@ pytest==4.1.1
 python-dateutil==2.7.5
 pytoml==0.1.19
 pytz==2018.5
-raiden-contracts==0.8.0
+raiden-contracts==0.9.0
```
Comment (LefterisJP) [inline constraints.txt]: Hmm I don't know if this is the only client change that will be needed with this version upgrade.  I suspect more places may be affected.

Comment (palango) [reply constraints.txt]: According to the changelog not: https://github.com/raiden-network/raiden-contracts/blob/master/CHANGELOG.md

But we'll see. First thing is a conflict with `raiden-libs`. So we have to wait for #3393 and #3400.

**Comment by LefterisJP** [comment]
Additionally I would qualify this as changelog entry-worthy.

File: constraints.txt (hunk 1 above)
Comment (LefterisJP) [reply constraints.txt]: Hehe .. as you saw there was more changes -- and this is just for what the tests cover.

File: raiden/settings.py [hunk 2]
```diff
@@ -44,3 +44,6 @@
 
 ORACLE_BLOCKNUMBER_DRIFT_TOLERANCE = 3
 ETHERSCAN_API = 'https://{network}.etherscan.io/api?module=proxy&action={action}'
+
+DEFAULT_LIMITS_CONTRACT_VERSION = '0.4.0'
```
Comment (LefterisJP) [inline raiden/settings.py]: Why call them default here? Wouldn't it make more sense to name these two as:

```python
RED_EYES_CONTRACT_VERSION = '0.4.0'
DEVELOPMENT_CONTRACT_VERSION = '0.3._'
```

Notice that `0.3._` also has limits.

Comment (palango) [reply raiden/settings.py]: Good point. Fixed.

**Pull request merged**